docs/validator/staking.md

```diff
@@ -146,23 +228,28 @@ The 0/0/40 shows the total validators: connected peers / up to date peers / my p
 
 To learn more about how validators are chosen, take a look at the [Staking FAQ](../validator/staking-faq.md).
 
-## See current list of Validators and stake amounts
+## See the list of current and future validators
 
-To see the current list of validators, you can take a look here: [http://rpc.betanet.near.org/status](http://rpc.betanet.near.org/status)
+You can use `near-cli` to review the validator set in the next three epochs:
 
-If you would like to see how much a validator is staking, you can run the command `near state <account name>` in Near CLI.
+| Command | Meaning |
+| -----| ----- |
+| `near proposals` | All validators that sent a staking transaction (`Proposal(Accepted)`), or are re-elected (`Rollover`) |
+| `near validators next` | All validators that will produce blocks in the next epoch. Can be `New`, `Rewarded` or `Kicked out` |
+| `near validators current` | All Validators that are producing blocks in the current epoch, and the number of block they produced |
 
-```bash
-{
-  amount: '100011163887239132720351',
-  code_hash: '11111111111111111111111111111111',
-  locked: '97985903901882082761',
-  storage_paid_at: 25,
-  storage_usage: 182
-}
-```
+<blockquote class="warning">
+    <strong>Heads Up!</strong><br><br>
+    You have to wait at least 2 epochs (43,200 blocks each) before your 
```

Comment:
@stefanopepe aren't we missing a word after "your"? The sentence seems to end abruptly.

Reply:
I wrote this one a while back, yes it's possible that on some edit I cut too much text. Feel free to submit a PR for this one!

Reply:
@stefanopepe here is an attempt to fix it: https://github.com/near/docs/pull/839. Let me know if it was about something else. Best regards